skimage/segmentation/_slic.pyx

```diff
@@ -200,18 +236,21 @@ def _slic_cython(np_floats[:, :, :, ::1] image_zyx,
 
 def _enforce_label_connectivity_cython(Py_ssize_t[:, :, ::1] segments,
                                        Py_ssize_t min_size,
-                                       Py_ssize_t max_size):
+                                       Py_ssize_t max_size,
+                                       Py_ssize_t start_label=1):
     """ Helper function to remove small disconnected regions from the labels
 
     Parameters
     ----------
-    segments : 3D array of int, shape (Z, Y, X)
+    segments: 3D array of int, shape (Z, Y, X)
         The label field/superpixels found by SLIC.
     min_size: int
         Minimum size of the segment
     max_size: int
         Maximum size of the segment. This is done for performance reasons,
         to pre-allocate a sufficiently large array for the breadth first search
+    start_label: int
+        label indexing start value.
```

Comment:
Full, capitalized sentence here, please, and some more clarity on what it means (I understand, but not sure other readers would).